api/python/t4/util.py

```diff
@@ -43,7 +43,15 @@
 # Used to satisfy search queries
 # elastic_search_url: https://example.com/es
 elastic_search_url:
-"""
+
+# default_local_registry: <url string, default: local appdirs>
+# default target registry for operations like install and build
+default_local_registry: {}
```

Comment:
Put it in quotes, just in case. I feel like URLs could cause problems in YAML... But they can't contain quotes, so this should be good enough.